docs/sphinx/examples/python/building_kernels.py

```diff
@@ -0,0 +1,266 @@
+# # Building Kernels
```

Comment:
What's the purpose of having both building_kernels.py and building_kernels,ipnyb?